examples/custom-field/3-pair-field-json/index.ts

```diff
@@ -93,7 +93,9 @@ export function pair<ListTypeInfo extends BaseListTypeInfo>(
           arg: graphql.arg({ type: PairInput }),
           // eslint-disable-next-line @typescript-eslint/no-unused-vars
           resolve(value, context) {
-            return resolveInput(value);
+            if (value !== undefined) {
+              return resolveInput(value);
+            }
```

Comment:
This should probably be in the `resolveInput` function

Reply:
I applied your suggestion and it works great (I thought initially this would break the create operation).